scripts/ci/pr-bot/shared/commentStrings.ts

```diff
@@ -70,3 +70,33 @@ export function reviewersAlreadyAssigned(reviewers: string[]): string {
 export function noLegalReviewers(): string {
   return "No reviewers could be found from any of the labels on the PR or in the fallback reviewers list. Check the config file to make sure reviewers are configured";
 }
+
+export function assignNewReviewer(labelToReviewerMapping: any): string {
+  let commentString =
+    "Assigning new set of reviewers because Pr has gone too long without review. If you would like to opt out of this review, comment `assign to next reviewer`:\n\n";
+
+  for (let label in labelToReviewerMapping) {
+    let reviewer = labelToReviewerMapping[label];
+    if (label == "no-matching-label") {
```

Comment:
Triple equals for string matching. Should `"no-matching-label"` be a reusable constant?